diff --git a/node_modules/vitepress-theme-teek/lib/components/theme/ArticleOverviewPage/src/index.vue2.js b/node_modules/vitepress-theme-teek/lib/components/theme/ArticleOverviewPage/src/index.vue2.js
index d179f03..3897a6d 100644
--- a/node_modules/vitepress-theme-teek/lib/components/theme/ArticleOverviewPage/src/index.vue2.js
+++ b/node_modules/vitepress-theme-teek/lib/components/theme/ArticleOverviewPage/src/index.vue2.js
@@ -37,7 +37,7 @@ var _sfc_main = /* @__PURE__ */ Vue.defineComponent({
     const categoriesPageLink = Vue.computed(() => {
       const localeIndexConst = localeIndex.value;
       const localeName = localeIndexConst !== "root" ? `/${localeIndexConst}` : "";
-      return `${localeName}${categoryConfig.value.path}${site.value.cleanUrls ? "" : ".html"}`;
+      return vitepress.withBase(`${localeName}${categoryConfig.value.path}${site.value.cleanUrls ? "" : ".html"}`);
     });
     const getFileWords = (url) => {
       return eachFileWords.value.filter((item) => {
